builder/builder.go

```diff
@@ -1311,6 +1312,102 @@ func (b *Builder) UpdateMixVer(version int) error {
 	return ioutil.WriteFile(filepath.Join(b.Config.Builder.VersionPath, b.MixVerFile), []byte(b.MixVer), 0644)
 }
 
+// UpdateOsRelease sets the mix version in the full-chroot os-release file
+func (b *Builder) UpdateOsRelease(version string) error {
```

Comment:
Why do you make this a method with a `*Builder` receiver? it only needs the statedir and the version to write to it. Seems a little heavy-weight to be passing the huge builder obj around (conceptually, I know it's just a pointer).